cmd/genswaggertypedocs/OWNERS

```diff
@@ -0,0 +1,3 @@
+reviewers:
+- david-mcmahon
+- luxas
```

Comment:
Can I opt out of this one please?
Don't know anything really about the swagger generation process